website/content/en/docs/building-operators/ansible/tutorial.md

```diff
@@ -18,7 +18,7 @@ please [migrate][migration-guide], or consult the [legacy docs][legacy-quickstar
   Replace it with another value if using a different registry or namespace.
   - The registry/namespace must be public, or the cluster must be provisioned with an
   [image pull secret][k8s-image-pull-sec] if the image namespace is private.
-
+- Ensure that you checked the [FAQ: Can I use the Kubebuilder docs?][faq-can-I-use-kb-docs].
```

Comment:
Again this is an awkward place, like we're forcing the issue. Nothing in this document makes me wonder if I can use kubebuilder.

```suggestion
```